### PR TITLE
Fix and clarify skybox texture order documentation

### PR DESCRIPTION
**Pull request by GreenXenith**
The order of skybox textures is currently listed as Y+ (top), Y- (bottom), X- (west), X+ (east), Z+ (north), Z- (south). Testing with simple T/B/W/E/N/S textures and comparing with F5 proves this is clearly not the case.

The original Irrlicht `CSkyBoxSceneNode` defines Z- as the "front" and a direction order of left/right/front/back, making the plane order X+/X-/Z-/Z+. The documentation was erroneously based on the direction order rather than the plane order.

Also, Irrlicht (now Minetest) aligns the top/bottom textures with the right (East) face (`─━┼─`), where most skybox textures are aligned with the front face (`─┿──`). I have clarified this accordingly.

Most people have probably used the order N/S/E/W, which effectively rotates the skybox such that the North texture is on the East face, which works fine as long as the texture doesn't need to be cardinally-aligned.

Aside: Don't make PRs when sleepy :D

### CODE REVIEW

**Review by appgurueu — APPROVED** [review]
Can confirm that the order is wrong and your order is correct.

I find it a bit hard to follow your notes on alignment, just by reading alone:

* What does it mean to be "aligned" with a plane? What I observe is that along the "shared axis" of the two planes, texture coordinates "align". I wonder whether there is a good way to put this into text.
* What does "front-aligned" mean? Aligned with the Z+ plane? Or aligned with the Z- plane?

(Still this has my approval, since it is definitely an improvement on the current wrong docs. In the end, modders will probably end up experimenting until it looks right anyways if they can't follow the explanation; explaining visuals in text is a bit difficult.)

**Comment by GreenXenith** [comment]
>     * What does it mean to be "aligned" with a plane? What I observe is that along the "shared axis" of the two planes, texture coordinates "align". I wonder whether there is a good way to put this into text.

"The top (+Y) and bottom (-Y) textures are oriented in-line with the -X (east) face. The top edge of the bottom texture and the bottom edge of the top texture touch the east face."

>     * What does "front-aligned" mean? Aligned with the Z+ plane? Or aligned with the Z- plane?

"Skybox textures expecting to be aligned with the north face ..."

Better, perhaps, @appgurueu?
